Defs/ThingDefs_Misc/Weapons_Turrets.xml

```diff
@@ -158,6 +218,7 @@
         <ammoSet>AmmoSet_20x102mmNATO</ammoSet>
       </li>
       <li Class="CombatExtended.CompProperties_FireModes">
+        <aimedBurstShotCount>5</aimedBurstShotCount>
```

Comment:
Auto turrets don't get burst fire modes--if they're capable of full auto, that's what they use. This inaccuracy and ammo consumption is an intended drawback of auto turrets vs manned ones.